server/src/main/java/com/google/coffeehouse/storagehandler/StorageHandler.java

```diff
@@ -71,7 +71,7 @@ public static Person getPerson(DatabaseClient dbClient, String userId) {
                                            .setNickname(row.getString(/* nicknameIndex= */1))
                                            .setUserId(userId);
 
-      if (!row.isNull(/* index= */2) || !row.getString(/* index= */2).isEmpty()) {
+      if (!row.isNull(/* index= */2)) {
```

Comment:
super nit: for this inline comment, there is no space between the variable and the equals sign, while below you have a space between. I believe the no space is the correct format, but I would choose one for styling consistency

Reply:
I will change to no space between the inline comments, thank you for catching this style issue!